src/Symfony/Component/Cache/Traits/FilesystemTrait.php

```diff
@@ -94,7 +94,7 @@ protected function doHave($id)
     protected function doSave(array $values, $lifetime)
     {
         $ok = true;
-        $expiresAt = time() + ($lifetime ?: 31557600); // 31557600s = 1 year
+        $expiresAt = $lifetime > 0 ? (time() + $lifetime) : 0;
```

Comment:
I'd suggest to write this as `$lifetime ? time() + $lifetime : 0;`

Reply:
… and done